client/lib/plans/features-list.js

```diff
@@ -250,12 +250,12 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_VIDEO_CDN_UNLIMITED ]: {
 		getSlug: () => constants.FEATURE_VIDEO_CDN_UNLIMITED,
-		getTitle: () => i18n.translate( 'Unlimited Video Storage' ),
+		getTitle: () => i18n.translate( 'Unlimited video storage' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.